src/test/java/seedu/duke/exercisegenerator/GenerateBodyExerciseTest.java

```diff
@@ -0,0 +1,31 @@
+package seedu.duke.exercisegenerator;
+
+import org.junit.jupiter.api.Test;
+import seedu.duke.exersisedata.ExerciseData;
+
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+public class GenerateBodyExerciseTest {
+    private static String BODY = "body only";
+
+    /**
+     * Checks if program accurately filters body weight exercises from
+     * entire list of data
+     */
```

Comment:
Can explain the test more i.e to say like explain the expected output

Reply:
ok, will update in next pr